script.xbmc.boblight/changelog.txt

```diff
@@ -0,0 +1,83 @@
+2.0.19
```

Comment:
Changelog is no longer parsed by kodi, it'd be better if you could use the `<news>` tag in addon.xml for the latest changes to the addon